types/benches/bytesrepr_bench.rs

```diff
@@ -437,6 +446,300 @@ fn deserialize_u512(b: &mut Bencher) {
     b.iter(|| U512::from_bytes(black_box(&num_u512_bytes)))
 }
 
+fn generate_account(associated_keys_len: u8, named_keys_len: u8) -> Account {
+    let account_hash = AccountHash::default();
+    let named_keys: NamedKeys = {
+        let mut tmp = BTreeMap::new();
+        (0..named_keys_len).for_each(|i| {
```

Comment:
It's used both in account generator and contract generator. You can reactor this closure into

```rust
fn sample_named_keys(n: usize) -> NamedKeys {}
```

Reply:
Done in [`b07c708` (#2389)](https://github.com/casper-network/casper-node/pull/2389/commits/b07c70853e7d9447e20fd5a17ad635a5b8606ad5).